pages/docs/reference/building-mpp-with-gradle.md

```diff
@@ -1295,23 +1295,22 @@ kotlin {
 </div>
 </div>
 
-### Experimental metadata publishing mode
+{:#experimental-metadata-publishing-mode}
+
+### Metadata publishing
 
 Gradle module metadata provides rich publishing and dependency resolution features that are used in Kotlin 
 multiplatform projects to simplify dependencies configuration for build authors. In particular, the publications of a 
 multiplatform library may include a special 'root' module that stands for the whole library and is automatically 
 resolved to the appropriate platform-specific artifacts when added as a dependency, as described below.
 
-In Gradle 5.3 and above, the module metadata is always used during dependency resolution, but publications don't 
-include any module metadata by default. To enable module metadata publishing, add 
-`enableFeaturePreview("GRADLE_METADATA")` to the root project's `settings.gradle` file. With older Gradle versions, 
-this is also required for module metadata consumption.
+In Gradle 6.0 and above, the module metadata is always used during dependency resolution and included in publications.
 
-> Note that the module metadata published by Gradle 5.3 and above cannot be read by Gradle versions older
-> than 5.3. 
-{:.note}
+In Gradle 5.3 and above, the module metadata is used during dependency resolution, but publications don't 
```

Comment:
Maybe replace `5.3 and above` with `versions below 6.0` or `versions 5.3 to 5.6.4`?

Reply:
Updated with I_n earlier Gradle versions starting from 5.3_